odoo/orm/models.py

```diff
@@ -562,6 +559,13 @@ class BaseModel(metaclass=MetaModel):
     _transient_max_hours = lazy_classproperty(lambda _: config.get('transient_age_limit'))
     "maximum idle lifetime (in hours), unlimited if ``0``"
 
+    id = Id(automatic=True)
+    display_name = Char(
```

Comment:
I don't really care but the automatic still exist at this commit (first commit). Then display_name should be automatic too.